### PR TITLE
[MavenV4] upgrade MavenV4 to node 20

### PR DESCRIPTION
**Pull request by qianz2**
**Task name**: MavenV4 

**Description**: update Node 20

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: _generated/MavenV4.versionmap.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Default|4.223.1
+Node20-219|4.223.0
```
Comment (qianz2) [inline _generated/MavenV4.versionmap.txt:2]: Is 219 the sprint number? 219 is for Node16, not sure if it's same as node 20?

**Comment by DmitriiBobreshev** [comment]
Hi @qianz2, Thank you for the PR, but for now, [the pipeline agent](https://github.com/microsoft/azure-pipelines-agent/blob/master/src/Agent.Worker/Handlers/NodeHandler.cs#L56) supports only Node 6, Node 10 and Node16 versions, so even if we add Node20 handler in task.json it won't work. For now you can transfer PR to draft and keep watching the agent releases, once the new handler will be added we could check the PR

**Comment by qianz2** [comment]
> Hi @qianz2, Thank you for the PR, but for now, [the pipeline agent](https://github.com/microsoft/azure-pipelines-agent/blob/master/src/Agent.Worker/Handlers/NodeHandler.cs#L56) supports only Node 6, Node 10 and Node16 versions, so even if we add Node20 handler in task.json it won't work. For now you can transfer PR to draft and keep watching the agent releases, once the new handler will be added we could check the PR

Hi @DmitriiBobreshev, thanks for pointing it out. I'm trying to add Node 20 to the infrastructure. To add handler for node 20 and test it locally, is the file you pointed out the only place where I need to add?

**Comment by qianz2** [comment]
PR for supporting node 20 in pipeline agent: https://github.com/microsoft/azure-pipelines-agent/pull/4344

File: _generated/MavenV4.versionmap.txt (hunk 1 above)
Comment (merlynomsft) [reply _generated/MavenV4.versionmap.txt:2]: Yes sprint number.  Please use current sprint instead of 219 (https://whatsprintis.it/)

**Pull request closed without merging**